include/NovelRT/Maths/GeoMatrix4x4F.h

```diff
@@ -22,139 +25,726 @@ namespace NovelRT::Maths
+         *      \end{bmatrix}
+         *      -6
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      -
+         *      \begin{bmatrix}
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1-6 & 7-6 & 6-6 & 5-6\\
+         *      2-6 & 1-6 & 4-6 & 3-6\\
+         *      3-6 & 5-6 & 1-6 & 2-6\\
+         *      4-6 & 6-6 & 7-6 & 1-6\\
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      -5 & 1 & 0 & -1\\
+         *      -4 & -5 & -2 & -3\\
+         *      -3 & -1 & -5 & -4\\
+         *      -2 & 0 & 1 & -5\\
+         *      \end{bmatrix}
+         * \f]
+         * @param other The other scalar that gets subtracted from this matrix.
+         * @return This matrix who's components consist of the entrywise difference of this and another scalar.
+         */
         inline GeoMatrix4x4F operator-=(float other) noexcept
         {
             *reinterpret_cast<glm::mat4*>(this) -= other;
             return *this;
         }
 
+        /**
+         * @brief Constructs a matrix containing the product of this and another scalar and applies it to this instance.
+         *
+         * @details
+         * An example of a matrix that would be constructed from multiplying a matrix by a scalar: \f[
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      \cdot 6
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      \cdot
+         *      \begin{bmatrix}
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6\\
+         *      6 & 6 & 6 & 6
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1*6+7*6+6*6+5*6 & 1*6+7*6+6*6+5*6 & 1*6+7*6+6*6+5*6 & 1*6+7*6+6*6+5*6\\
+         *      2*6+1*6+4*6+3*6 & 2*6+1*6+4*6+3*6 & 2*6+1*6+4*6+3*6 & 2*6+1*6+4*6+3*6\\
+         *      3*6+5*6+1*6+2*6 & 3*6+5*6+1*6+2*6 & 3*6+5*6+1*6+2*6 & 3*6+5*6+1*6+2*6\\
+         *      4*6+6*6+7*6+1*6 & 4*6+6*6+7*6+1*6 & 4*6+6*6+7*6+1*6 & 4*6+6*6+7*6+1*6
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      114 & 114 & 114 & 114\\
+         *      60 & 60 & 60 & 60\\
+         *      66 & 66 & 66 & 66\\
+         *      108 & 108 & 108 & 108
+         *      \end{bmatrix}
+         * \f]
```

Comment:
As per my other question, is this correct?

Reply:
after testing it was not correct and has been changed to be correct.